### ISSUE TITLE
NTR: Quinolinic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8652]
New term request Quinolinic acid high in urine (elthat_200523160452):   UMLS: 5394148

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000335
- label: Elevated urinary quinolinic acid level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: PMID:29517768
- def: The amount of quinolinic acid in the urine, normalized for urine concentration, is above the upper limit of normal. Quinolinic acids is a pyridinedicarboxylic acid that is pyridine substituted by carboxy groups at positions 2 and 3. It is a metabolite of tryptophan.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8652
- synonyms:
